examples/Rudaina_work_area/genzsprtwo_21_travel_time.py

```diff
@@ -0,0 +1,48 @@
+import requests
+
+"""
+
+origin: user input of origin place
+destination: user input of distination
+return: estimated travel time between the 2 destnatiosn
+"""
+def get_distance_and_duration(origin, destination):
+    params = {
+        'origins': origin,
+        'destinations': destination,
+        'key': "Enter API KEY HERE",
+        'units': 'metric' 
+    }
+
+    try:
+        response = requests.get("https://maps.googleapis.com/maps/api/distancematrix/json?", params=params)
+        data = response.json()
+
+        if data['status'] == 'OK':
+            for row in data['rows']:
+                for element in row['elements']:
+                    distance_text = element['distance']['text']
+                    duration_text = element['duration']['text']
+                    return distance_text, duration_text
```

Comment:
I believe we can directly access the first element since exception handling is also done